tldw_Server_API/app/core/Ingestion_Media_Processing/Video/Video_DL_Ingestion_Lib.py

```
@@ -1164,8 +1214,24 @@ def process_single_video(
     local_file_path_for_transcription = None
     # Temp dir for download is provided by the caller (`temp_dir`)
 
+    def _is_cancelled() -> bool:
+        """Return True if cancellation is requested, logging callback errors."""
+        if cancel_check is None:
+            return False
+        try:
+            return bool(cancel_check())
+        except Exception as exc:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)